pkgs/development/tools/java/jprofiler/default.nix

```
@@ -1,45 +1,45 @@
 { stdenv
 , lib
+, callPackages
 , fetchurl
 , makeWrapper
 , makeDesktopItem
 , copyDesktopItems
-, undmg
+, autoPatchelfHook
+, callPackage
 , jdk
+, libX11
+, musl
+, component ? "ui"
 }:
 
 let
-  inherit (stdenv.hostPlatform) system;
-  pname = "jprofiler";
+  versions = import ./version.nix;
```

Since you've changed the name, you should also change the filename.
```suggestion
  versions = import ./versions.nix;
```

---

I guess I've found much better naming: artefacts.